content/blog/2019-01-29-rdhs.md

```diff
@@ -0,0 +1,580 @@
+## 1 /home/oj/GoogleDrive/AcademicWork/Imperial/git/rdhs/paper/project_one/datasets/CDPR61FL.rds
+## 2 /home/oj/GoogleDrive/AcademicWork/Imperial/git/rdhs/paper/project_one/datasets/CDPR61FL.rds
+## 3 /home/oj/GoogleDrive/AcademicWork/Imperial/git/rdhs/paper/project_one/datasets/TZPR7AFL.rds
+## 4 /home/oj/GoogleDrive/AcademicWork/Imperial/git/rdhs/paper/project_one/datasets/TZPR7AFL.rds
+##   survey_id
+## 1 CD2013DHS
+## 2 CD2013DHS
+## 3 TZ2015DHS
+## 4 TZ2015DHS
+```
+More information about download options and querying the survey questions can be found 
+[here](https://ropensci.github.io/rdhs/articles/introduction.html#load-datasets-and-associated-metadata-into-r-)
+
+#### 5. Extract variables and combine datasets
+
+To extract our data we pass our questions object to the function `extract_dhs`, 
+which will create a list with each dataset and its extracted data as a `data.frame`. 
+
+```r
+# extract the data and add geographic information too
+extract <- extract_dhs(questions, add_geo = FALSE)
+```
+
+The resultant extract is a list, with a new element for each different dataset 
+that you have extracted. We can now combine our two data frames for further analysis using the `rdhs` package 
+function `rbind_labelled()`:
+
+```r
+# first let's bind our first extraction, without the hv024
+extract_bound <- rbind_labelled(extract)
+```
+
+```
+## Warning in rbind_labelled(extract): Some variables have non-matching value labels: hv024.
+## Inheriting labels from first data frame with labels.
+```
+
+The thrown warning has shown us that *hv024* did not have matching labels between 
+the two lists, and the labels from the first list have been used. 
+*hv024* stores the regions for these 2 countries, and we probably want to keep all 
+the labels, which we can do by using the `labels` argument:
+
+
+```r
+# lets try concatenating the hv024
+better_bound <- rbind_labelled(extract, labels = list("hv024"="concatenate"))
+```
+We could also specify new labels for a variable. For example, imagine the two 
+datasets encoded their RDT responses differently, with the first one as 
+`c("No","Yes")` and the other as `c("Negative","Positive")`. We can choose to
+relabel these, e.g. as `c("NegativeTest","PositiveTest")`:
+
+
+```r
+# lets try concatenating the hv024 and providing new labels
+better_bound <- rbind_labelled(
+  extract,
+  labels = list("hv024"="concatenate",
+                "hml35"=c("NegativeTest"=0, "PositiveTest"=1))
+)
+
+# and our new label
+head(attr(better_bound$hml35,"labels"))
+```
+
+```
+## NegativeTest PositiveTest 
+##            0            1
+```
+
+For more information about controlling how to extract data from your downloaded
+sections, see the [last section in the introduction vignette](https://ropensci.github.io/rdhs/articles/introduction.html#extract-variables-and-combining-datasets-for-pooled-multi-survey-analyses-). 
+
+---
+
+We now have managed to go from our initial request for data about the use of 
```

Comment:
Can you add 1-2 sentences saying more specifically what you now have in the "a finalised data set" after these 5 steps? The plot in step 1 lures the reader with a really interesting research question, but after focusing on the technical steps, it's hard to know what we end up with. 

For the record, I'm realizing this is one of the most common things I see in draft blog posts about packages.

Reply:
I've added a bit more here and have tried to reframe the end of the package overview, to try and remind the reader why we are interested in building these data sets in the first place. Thanks for picking this up.

Reply:
this is great thanks